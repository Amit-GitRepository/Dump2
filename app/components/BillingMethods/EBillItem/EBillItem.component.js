/* EBillItem: show details of e-bill item such as ebill option, productType etc. */

import camelCase from 'lodash/camelCase';
import noop from 'lodash/noop';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './EBillItem.style';
import {Button, ISText} from '../../Shared';
import {translate} from '../../../language/i18n/helper';
import {View} from 'react-native';

class EBillItem extends Component {

  generateEBillOption = () => {
    const {billingFormat, ebillValue, onEditBill} = this.props;
    return (
      <View style={styles.ebillOptionContainer}>
        <ISText type='BOLD' style={styles.ebillOptionFormat}>
          {translate(`BILLING_METHODS__${billingFormat.toUpperCase()}`)}
        </ISText>
        <ISText style={styles.ebillOptionValue}>
          {ebillValue}
        </ISText>
        <Button
          textType='SEMIBOLD'
          type='inline'
          text={translate('BILLING_METHODS__EDIT')}
          onPress={onEditBill}
        />
      </View>
    );
  }

  render () {
    const {productId, accountType, noBorder} = this.props;
    return (
      <View style={[styles.container, noBorder ? null : styles.borderBottom]}>
        <View style={styles.serviceInfoContainer}>
          <ISText type='BOLD' style={styles.serviceNumber}>{productId}</ISText>
          <ISText type='BOLD' style={styles[camelCase(accountType)]}>{accountType}</ISText>
        </View>
        {this.generateEBillOption()}
      </View>
    );
  }
}

EBillItem.defaultProps = {
  productId: '',
  accountType: '',
  onEditBill: noop,
  noBorder: false
};

EBillItem.propTypes = {
  accountType: PropTypes.string.isRequired,
  productId: PropTypes.string.isRequired,
  billingFormat: PropTypes.string,
  ebillValue: PropTypes.string,
  onEditBill: PropTypes.func,
  noBorder: PropTypes.bool
};

export default EBillItem;
