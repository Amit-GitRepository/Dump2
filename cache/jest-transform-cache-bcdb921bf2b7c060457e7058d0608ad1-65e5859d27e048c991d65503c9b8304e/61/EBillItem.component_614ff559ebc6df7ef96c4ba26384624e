Object.defineProperty(exports,"__esModule",{value:true});var cov_2adols6y20=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillingMethods/EBillItem/EBillItem.component.js',hash='d9d011a0086fcd4f5b672fc9cda3cf2b32037385',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillingMethods/EBillItem/EBillItem.component.js',statementMap:{'0':{start:{line:15,column:50},end:{line:15,column:60}},'1':{start:{line:16,column:4},end:{line:31,column:6}},'2':{start:{line:35,column:47},end:{line:35,column:57}},'3':{start:{line:36,column:4},end:{line:44,column:6}},'4':{start:{line:48,column:0},end:{line:53,column:2}},'5':{start:{line:55,column:0},end:{line:62,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:14,column:24},end:{line:14,column:25}},loc:{start:{line:14,column:30},end:{line:32,column:3}},line:14},'1':{name:'(anonymous_1)',decl:{start:{line:34,column:2},end:{line:34,column:3}},loc:{start:{line:34,column:12},end:{line:45,column:3}},line:34}},branchMap:{'0':{loc:{start:{line:37,column:38},end:{line:37,column:75}},type:'cond-expr',locations:[{start:{line:37,column:49},end:{line:37,column:53}},{start:{line:37,column:56},end:{line:37,column:75}}],line:37}},s:{'0':0,'1':0,'2':0,'3':0,'4':0,'5':0},f:{'0':0,'1':0},b:{'0':[0,0]},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();

var _camelCase=require('lodash/camelCase');var _camelCase2=_interopRequireDefault(_camelCase);
var _noop=require('lodash/noop');var _noop2=_interopRequireDefault(_noop);
var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _EBillItem=require('./EBillItem.style');var _EBillItem2=_interopRequireDefault(_EBillItem);
var _Shared=require('../../Shared');
var _helper=require('../../../language/i18n/helper');
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

EBillItem=function(_Component){_inherits(EBillItem,_Component);function EBillItem(){var _ref;var _temp,_this,_ret;_classCallCheck(this,EBillItem);for(var _len=arguments.length,args=Array(_len),_key=0;_key<_len;_key++){args[_key]=arguments[_key];}return _ret=(_temp=(_this=_possibleConstructorReturn(this,(_ref=EBillItem.__proto__||Object.getPrototypeOf(EBillItem)).call.apply(_ref,[this].concat(args))),_this),_this.

generateEBillOption=function(){cov_2adols6y20.f[0]++;var _ref2=(cov_2adols6y20.s[0]++,
_this.props),ebillFormat=_ref2.ebillFormat,ebillValue=_ref2.ebillValue,onEditBill=_ref2.onEditBill;cov_2adols6y20.s[1]++;
return(
_react2.default.createElement(_reactNative.View,{style:_EBillItem2.default.ebillOptionContainer},
_react2.default.createElement(_Shared.ISText,{type:'BOLD',style:_EBillItem2.default.ebillOptionFormat},
(0,_helper.translate)('BILLING_METHODS__'+ebillFormat.toUpperCase())),

_react2.default.createElement(_Shared.ISText,{style:_EBillItem2.default.ebillOptionValue},
ebillValue),

_react2.default.createElement(_Shared.Button,{
textType:'SEMIBOLD',
type:'inline',
text:(0,_helper.translate)('BILLING_METHODS__EDIT'),
onPress:onEditBill})));



},_temp),_possibleConstructorReturn(_this,_ret);}_createClass(EBillItem,[{key:'render',value:function render()

{cov_2adols6y20.f[1]++;var _ref3=(cov_2adols6y20.s[2]++,
this.props),serviceID=_ref3.serviceID,accountType=_ref3.accountType,noBorder=_ref3.noBorder;cov_2adols6y20.s[3]++;
return(
_react2.default.createElement(_reactNative.View,{style:[_EBillItem2.default.container,noBorder?(cov_2adols6y20.b[0][0]++,null):(cov_2adols6y20.b[0][1]++,_EBillItem2.default.borderBottom)]},
_react2.default.createElement(_reactNative.View,{style:_EBillItem2.default.serviceInfoContainer},
_react2.default.createElement(_Shared.ISText,{type:'BOLD',style:_EBillItem2.default.serviceNumber},serviceID),
_react2.default.createElement(_Shared.ISText,{type:'BOLD',style:_EBillItem2.default[(0,_camelCase2.default)(accountType)]},accountType)),

this.generateEBillOption()));


}}]);return EBillItem;}(_react.Component);cov_2adols6y20.s[4]++;


EBillItem.defaultProps={
serviceID:'',
accountType:'',
onEditBill:_noop2.default,
noBorder:false};cov_2adols6y20.s[5]++;


EBillItem.propTypes={
accountType:_propTypes2.default.string.isRequired,
serviceID:_propTypes2.default.string.isRequired,
ebillFormat:_propTypes2.default.string,
ebillValue:_propTypes2.default.string,
onEditBill:_propTypes2.default.func,
noBorder:_propTypes2.default.bool};exports.default=


EBillItem;