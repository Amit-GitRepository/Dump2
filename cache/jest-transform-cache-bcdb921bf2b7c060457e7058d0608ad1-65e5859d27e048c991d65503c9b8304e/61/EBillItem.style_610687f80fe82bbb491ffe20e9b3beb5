Object.defineProperty(exports,"__esModule",{value:true});var cov_1ouc2glo2e=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillingMethods/EBillItem/EBillItem.style.js',hash='d7072df26eac5683a79fcdd62277d11f4a6ea16a',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillingMethods/EBillItem/EBillItem.style.js',statementMap:{},fnMap:{},branchMap:{},s:{},f:{},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _constants=require('../../../themes/constants.styles');exports.default=

{
container:{
padding:15,
flex:1},

borderBottom:{
borderBottomWidth:1,
borderBottomColor:_constants.colors.PRIMARY_BG_SEPARATOR},

serviceInfoContainer:{
marginBottom:5},

serviceNumber:{
fontSize:_constants.fonts.FONT_SIZE_LARGE},

trueMoveHPostpaid:{
color:_constants.colors.SECONDARY_TMOVE},

trueMoveHPrepaid:{
color:_constants.colors.SECONDARY_TMOVE},

trueOnline:{
color:_constants.colors.SECONDARY_TONLINE},

trueVision:{
color:_constants.colors.SECONDARY_TVISION},

ebillOptionContainer:{
flexDirection:'row',
justifyContent:'space-between'},

ebillOptionFormat:{
fontSize:_constants.fonts.FONT_SIZE_MEDIUM,
paddingTop:4},

ebillOptionValue:{
fontSize:_constants.fonts.FONT_SIZE_NORMAL,
paddingTop:4}};