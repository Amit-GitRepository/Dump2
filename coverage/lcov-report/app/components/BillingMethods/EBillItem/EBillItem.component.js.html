<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for app/components/BillingMethods/EBillItem/EBillItem.component.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">app/components/BillingMethods/EBillItem</a> EBillItem.component.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>6/6</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>6/6</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">/* EBillItem: show details of e-bill item such as ebill option, productType etc. */
&nbsp;
import Button from '../../Shared/Button/Button.component';
import camelCase from 'lodash/camelCase';
import ISText from '../../Shared/ISText/ISText.component';
import noop from 'lodash/noop';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './EBillItem.style';
import {translate} from '../../../language/i18n/helper';
import {View} from 'react-native';
&nbsp;
class EBillItem extends Component {
&nbsp;
  generateEBillOption = () =&gt; {
    const {ebillFormat, ebillValue, onEditBill} = this.props;
    return (
      &lt;View style={styles.ebillOptionContainer}&gt;
        &lt;ISText type='BOLD' style={styles.ebillOptionFormat}&gt;
          {translate(`BILLING_METHODS__${ebillFormat}`)}
        &lt;/ISText&gt;
        &lt;ISText style={styles.ebillOptionValue}&gt;
          {ebillValue}
        &lt;/ISText&gt;
        &lt;Button
          textType='SEMIBOLD'
          type='inline'
          text={translate('BILLING_METHODS__EDIT')}
          onPress={onEditBill}
        /&gt;
      &lt;/View&gt;
    );
  }
&nbsp;
  render () {
    const {serviceID, accountType, noBorder} = this.props;
    return (
      &lt;View style={[styles.container, noBorder ? null : styles.borderBottom]}&gt;
        &lt;View style={styles.serviceInfoContainer}&gt;
          &lt;ISText type='BOLD' style={styles.serviceNumber}&gt;{serviceID}&lt;/ISText&gt;
          &lt;ISText type='BOLD' style={styles[camelCase(accountType)]}&gt;{accountType}&lt;/ISText&gt;
        &lt;/View&gt;
        {this.generateEBillOption()}
      &lt;/View&gt;
    );
  }
}
&nbsp;
EBillItem.defaultProps = {
  serviceID: '',
  accountType: '',
  onEditBill: noop,
  noBorder: false
};
&nbsp;
EBillItem.propTypes = {
  accountType: PropTypes.string.isRequired,
  serviceID: PropTypes.string.isRequired,
  ebillFormat: PropTypes.string,
  ebillValue: PropTypes.string,
  onEditBill: PropTypes.func,
  noBorder: PropTypes.bool
};
&nbsp;
export default EBillItem;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Oct 31 2017 11:24:42 GMT+0530 (IST)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
