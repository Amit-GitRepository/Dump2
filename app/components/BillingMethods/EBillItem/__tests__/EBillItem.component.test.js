import EBillItem from '../EBillItem.component';
import React from 'react';
import renderer from 'react-test-renderer'; // Enzyme adapter for React 16

describe('EBillItem component', () => {
  it('renders correctly with SMS option', () => {
    const component = renderer.create(<EBillItem
      accountType='True Vision'
      productId='9876543210'
      billingFormat='SMS'
      ebillValue='9876543210'
    />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
