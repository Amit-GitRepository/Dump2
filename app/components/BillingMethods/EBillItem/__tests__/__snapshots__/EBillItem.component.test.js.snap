// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EBillItem component renders correctly with SMS option 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "padding": 15,
      },
      Object {
        "borderBottomColor": "#E9E9EA",
        "borderBottomWidth": 1,
      },
    ]
  }
>
  <View
    style={
      Object {
        "marginBottom": 5,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#303C46",
          "fontFamily": "SukhumvitSet-Bold",
          "fontSize": 22,
        }
      }
    >
      9876543210
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#E82E8A",
          "fontFamily": "SukhumvitSet-Bold",
        }
      }
    >
      True Vision
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#303C46",
          "fontFamily": "SukhumvitSet-Bold",
          "fontSize": 18,
          "paddingTop": 4,
        }
      }
    >
      SMS
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#303C46",
          "fontFamily": "SukhumvitSet-Text",
          "fontSize": 16,
          "paddingTop": 4,
        }
      }
    >
      9876543210
    </Text>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "transparent",
          "borderRadius": 50,
          "borderWidth": 0,
          "opacity": 1,
          "padding": 5,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "stretch",
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "color": "#303C46",
                },
                Array [
                  Object {
                    "color": "#F93E60",
                    "fontSize": 16,
                  },
                  Object {},
                ],
                Object {
                  "fontFamily": "SukhumvitSet-SemiBold",
                },
              ]
            }
          >
            Edit
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
